src/import-page/import-stepper/ImportStepper.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect } from 'react';
+import {
+  injectIntl,
+  intlShape,
+} from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { Button } from '@edx/paragon';
+import { history } from '@edx/frontend-platform';
+
+import { getFormattedSuccessDate } from '../../export-page/utils';
+import { RequestStatus } from '../../data/constants';
+import CourseStepper from '../../generic/course-stepper';
+import { IMPORT_STAGES } from '../data/constants';
+import { fetchImportStatus } from '../data/thunks';
+import {
+  getCurrentStage, getError, getFileName, getLoadingStatus, getProgress, getSavingStatus, getSuccessDate,
+} from '../data/selectors';
+import messages from './messages';
+
+const ImportStepper = ({ intl, courseId }) => {
+  const currentStage = useSelector(getCurrentStage);
+  const fileName = useSelector(getFileName);
+  const { hasError, message: errorMessage } = useSelector(getError);
+  const progress = useSelector(getProgress);
+  const dispatch = useDispatch();
+  const loadingStatus = useSelector(getLoadingStatus);
+  const savingStatus = useSelector(getSavingStatus);
+  const successDate = useSelector(getSuccessDate);
+  const isStopFetching = currentStage === IMPORT_STAGES.SUCCESS
+    || loadingStatus === RequestStatus.FAILED
+    || savingStatus === RequestStatus.FAILED
+    || hasError;
+  const formattedErrorMessage = hasError ? errorMessage || intl.formatMessage(messages.defaultErrorMessage) : '';
+
+  useEffect(() => {
+    const id = setInterval(() => {
+      if (isStopFetching) {
+        clearInterval(id);
+      } else if (fileName) {
+        dispatch(fetchImportStatus(courseId, fileName));
+      }
+    }, 3000);
+    return () => clearInterval(id);
+  });
+
+  let successTitle = intl.formatMessage(messages.stepperSuccessTitle);
+  const formattedSuccessDate = getFormattedSuccessDate(successDate);
+  if (currentStage === IMPORT_STAGES.SUCCESS && formattedSuccessDate) {
+    successTitle += formattedSuccessDate;
+  }
+
+  const handleRedirectCourseOutline = () => history.push(`/course/${courseId}/outline`);
+
```

Comment:
Make this redirect to the legacy studio page. The legacy page will determine if they are taken to the MFE version or legacy version of the outline page.